app/views/due_soon_and_overdue_defects_mailer/notify.html.haml

```diff
@@ -0,0 +1,14 @@
+\# #{t('app.title')}
+\
+= I18n.t('email.defects.due_soon_and_overdue.heading')
+\
+- @defects.each do |defect|
```

Comment:
How does Notify cope with these line breaks? In my experience, I've had to escape the line breaks, so Notify doesn't read them as paragraphs.

Reply:
These are meant to be treated as paragraphs. I copied the formatting from other mailers in the system and tested to make sure the format came through (it does).

Reply:
Ah sweet, I thought that originally, then looked at something further down the line that made me think they might not have been 😄